lib/logstash/inputs/elasticsearch.rb

```diff
@@ -404,10 +393,9 @@ def validate_authentication
   def setup_client_ssl
     ssl_options = {}
     ssl_options[:ssl] = true if @ssl_enabled
+    ssl_options[:trust_strategy] = trust_strategy_for_ca_trusted_fingerprint
```

Comment:
oops, i should only set this if `@ssl_enabled`. I'll move that back to where it was before.